packages/@o3r/rules-engine/src/engine/engine.interface.ts

```diff
@@ -74,7 +74,16 @@ export interface EngineRule extends Rule {
 export interface EngineRuleset {
   /** Optional date range where the ruleset will be executed, it supports a dateString or a timestamp as number, more info on javascript Date() documentation */
   validityRange?: { from?: string | number; to?: string | number };
-  /** Component linked to the ruleset, if set it will disable the ruleset execution per default, waiting to a subscription */
+  /**
+   * Components linked to the ruleset. If present the ruleset will not be active by default.
+   * 'or' condition: If at least one component has subscribed, the ruleset will become active.
+   * If present, the {@link linkedComponent} property will not be taken into consideration
+   */
+  linkedComponents?: {or: { library: string; name: string }[]};
```

Comment:
We have this type `{ library: string; name: string }` at a lot of place in Otter Framework 
We may open an issue to create a reusable type no ?

Reply:
Good point . Either an issue or I could do it in this PR. It depends in how many places and packages it is used.